test/bin/mdx-test/expect/parts-begin-end/test-case.md

```diff
@@ -33,3 +33,17 @@ val x : int = 2
 
 ```ocaml file=parts-begin-end.ml,part=indented
 ```
+
+```ocaml
+# s ;;
+```
+
+```ocaml file=parts-begin-end.ml,part=toto
+```
+
+```ocaml
+# s ;;
+```
+
+```ocaml skip,file=parts-begin-end.ml,part=toto
+```
```

Comment:
Either this addition to the test is removed (which is fine since you added another test), or we add `skip` to the previous inclusions.

Reply:
I removed that part.